### DISCUSSION TITLE
Is there anyway to get the ability to add a photo using the mobile site for an asset?

### DISCUSSION THREAD

**Discussion by voron69-bit** [sysadminsmedia/homebox#213] [Ideas]
I can switch it to use the desktop site, but it is very small and difficult to read.

**Comment by superdong69** [comment]
Why isn't this already a feature? Can we get some information on if this is in the road map,  a known feature that is being worked on etc? Seems like a pretty benign update to get a 100x improvement on functionality.

**Comment by tankerkiller125** [comment]
This feature already exists, but it seems that the button is glitched on mobile:
![image](https://github.com/user-attachments/assets/08a0f565-97f6-46b3-86c8-03a894c312b7)


Related to #226

**Reply by voron69-bit** [reply]
I see! Is that something we can expect to be resolved in the not too distant future?

**Reply by tonyaellie** [reply]
https://github.com/sysadminsmedia/homebox/pull/229

**Reply by voron69-bit** [reply]
> #229

Any idea when this will go live into the live release?

**Reply by mcarbonne** [reply]
@voron69-bit if you don't want to wait for the next release, use ` ghcr.io/sysadminsmedia/homebox:main` as image.

**Reply by voron69-bit** [reply]
> @voron69-bit if you don't want to wait for the next release, use ` ghcr.io/sysadminsmedia/homebox:main` as image.

Unfortunately, I use the Unraid docker app.

**Reply by voron69-bit** [reply]
I changed the branch to main, and it installed, but the photo button on creating a new item is completely missing.

**Reply by voron69-bit** [reply]
Apparently it took a minute to sync and give me the option to update. I thought it would pull it down immediately on the recompose. It's there! Thank you guys for the hard work!
